emrun.py

```diff
@@ -738,13 +738,13 @@ def do_POST(self):
     self.wfile.write(b'OK')
 
 
-# Returns stdout by running command with universal_newlines=True
-def check_output(cmd, universal_newlines=True, *args, **kwargs):
+# Returns stdout by running command with text=True
+def check_output(cmd, text=True, *args, **kwargs):
   if hasattr(subprocess, "run"):
```

Comment:
You can delete this check, it's only needed to support Python < 3.5.

Reply:
Thanks for spotting that!

Reply:
Oh wait, sorry I forgot that emrun.py is the exception and it need to run on older python versions :(

Reply:
Reverted emrun.py part of the change.